.gitignore

```diff
@@ -14,3 +14,5 @@ compile_commands.json*
 .vscode/*
 **/build/*
 .vs/*
+
+make
```

Comment:
Is there a reason you added this?

Reply:
git add * would always add the "make" file, which doesn't make sense since it's a generated file. So i added it to the ignore list.